iaso/api/storage.py

```diff
@@ -349,7 +349,8 @@ class StorageLogViewSet(CreateModelMixin, viewsets.GenericViewSet):
 
     def create(self, request):
         """
-        POST /api/mobile/storage/logs
+        POST /api/mobile/storage/logs [Deprecated] will be removed in the future
```

Comment:
Nitpicking here but everywhere else you put the deprecated endpoint under the new one and not above like here